CMakeLists.txt

```diff
@@ -335,6 +342,20 @@ if (USE_GNUTLS)
 		target_link_libraries(datachannel PRIVATE Nettle::Nettle)
 		target_link_libraries(datachannel-static PRIVATE Nettle::Nettle)
 	endif()
+elseif(USE_MBEDTLS)
+	if(USE_SYSTEM_MBEDTLS)
+		find_package(MbedTLS 3 REQUIRED)
+	else()
+		file(READ deps/mbedtls/include/mbedtls/mbedtls_config.h MBEDTLS_CONFIG)
+		string(REPLACE "//#define MBEDTLS_SSL_DTLS_SRTP" "#define MBEDTLS_SSL_DTLS_SRTP" MBEDTLS_CONFIG "${MBEDTLS_CONFIG}")
+		file(WRITE deps/mbedtls/include/mbedtls/mbedtls_config.h "${MBEDTLS_CONFIG}")
+		add_subdirectory(deps/mbedtls EXCLUDE_FROM_ALL)
+	endif()
+
+	target_compile_definitions(datachannel PRIVATE USE_MBEDTLS)
+	target_compile_definitions(datachannel-static PRIVATE USE_MBEDTLS)
+	target_link_libraries(datachannel PRIVATE MbedTLS::mbedtls)
+	target_link_libraries(datachannel-static PRIVATE MbedTLS::mbedtls)
```

Comment:
The Finder provides `MbedTLS::MbedTLS` and not `MbedTLS::mbedtls`.

Reply:
@tytan652 That breaks the build

```
CMake Error at CMakeLists.txt:357 (target_link_libraries):
  Target "datachannel" links to:

    MbedTLS::MbedTLS

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at CMakeLists.txt:358 (target_link_libraries):
  Target "datachannel-static" links to:

    MbedTLS::MbedTLS

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.
```

Reply:
Because the submodules and the Finder for system install set different targets. So you need to use `MbedTLS::mbedtls` if `USE_SYSTEM_MBEDTLS=OFF` and `MbedTLS::MbedTLS` if `USE_SYSTEM_MBEDTLS=ON`.

Reply:
Fixed! Mbed TLS is not a submodule anymore, and use `MbedTLS::MbedTLS` properly.